lib/manager/maven/extract.ts

```diff
@@ -310,18 +365,43 @@ export async function extractAllPackageFiles(
   packageFiles: string[]
 ): Promise<PackageFile[]> {
   const packages: PackageFile[] = [];
+  const additionalRegistryUrls = [];
+
   for (const packageFile of packageFiles) {
     const content = await readLocalFile(packageFile, 'utf8');
     if (content) {
```

Comment:
can we invert the condition to have less code indention?

Reply:
I can do that.